hack/hooks/pre-commit

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
I'm not sure how to check file permissions in github, but can you make sure this is executable? The hook will be ignored if it isn't.

Reply:
```shell
$ ls -l hack/hooks/pre-commit
-rwxr-xr-x  1 dlorenc  eng  34 Oct 27 09:06 hack/hooks/pre-commit
```